zulipterminal/ui_tools/boxes.py

```diff
@@ -631,6 +631,22 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                 header.focus_col = self.FOCUS_HEADER_BOX_STREAM
             else:
                 header.focus_col = self.FOCUS_HEADER_BOX_RECIPIENT
+        elif is_command_key('AUTOCOMPLETE', key) and hasattr(
+                            self, 'msg_write_box'):
```

Comment:
Is this to handle the case of the compose box not being used yet?

Reply:
Yes, because some test cases test `autocomplete` without initiating `msg_write_box`, so I think that could happen in some scenarios based on these tests. That's why I made this check.